framework/rendering/postprocessing_renderpass.cpp

```diff
@@ -429,7 +446,13 @@ void PostProcessingRenderPass::prepare_draw(CommandBuffer &command_buffer, Rende
 			if (const uint32_t *sampled_attachment = it.second.get_target_attachment())
 			{
 				auto *image_rt = it.second.get_render_target();
-				sampled_attachments.insert({image_rt, *sampled_attachment});
+				auto  packed_sampled_attachment = *sampled_attachment;
+
+				// pack sampled attachment
+				if (it.second.is_depth_resolve())
+					packed_sampled_attachment |= 1 << 31;
```

Comment:
I'm not a huge fan of hijacking bits to convey extra information, but if we have to could we use some named constants for the bit and the mask rather than magic numbers (0x7FFFFFFF, 0x80000000, 1 << 31)?

Reply:
> I'm not a huge fan of hijacking bits to convey extra information, but if we have to could we use some named constants for the bit and the mask rather than magic numbers (0x7FFFFFFF, 0x80000000, 1 << 31)?

Fixed。